api/sql/schema/750$transfer.push.confirmMerchant.sql

```diff
@@ -14,3 +17,17 @@ WHERE
     merchantTxState = 1
 
 IF @@ROWCOUNT <> 1 RAISERROR('transfer.confirmMerchant', 16, 1);
+
```

Comment:
raise should be last statement, as we do not want to lose the event
see pattern in other similar procedures